src/site/layouts/event.drupal.liquid

```diff
@@ -77,105 +77,130 @@
+                      <span>{{ start_date_full }}
+                        -</span><br>
+                      <span>{{ end_date_full }}</span>
+                    {% endif %}
+                  {% endif %}
+                </dd>
+              </dl>
 
-                            <dl>
-                                <dt class="vads-u-font-weight--bold">Where:</dt>
-                                <dd>
-                                    <p class="vads-u-margin--0"><a href="{{ facility.entityUrl.path }}">{{ facility.title }}</a></p>
-                                    <p class="vads-u-margin--0">{{ fieldLocationHumanreadable }}</p>
-                                </dd>
+              <dl>
+                <dt class="vads-u-font-weight--bold">Where:</dt>
+                <dd>
+                  <p class="vads-u-margin--0">
+                    <a href="{{ facility.entityUrl.path }}">{{ facility.title }}</a>
+                  </p>
+                  <p class="vads-u-margin--0">{{ fieldLocationHumanreadable }}</p>
+                </dd>
 
-                            </dl>
+              </dl>
 
-                            {% if fieldEventCost %}
-                                <dl>
-                                    <dt class="vads-u-font-weight--bold">Cost:</dt>
-                                    <dd><span>{{ fieldEventCost }}</span></dd>
-                                </dl>
-                            {% endif %}
-                        </div>
+              {% if fieldEventCost %}
+                <dl>
+                  <dt class="vads-u-font-weight--bold">Cost:</dt>
+                  <dd>
+                    <span>{{ fieldEventCost }}</span>
+                  </dd>
+                </dl>
+              {% endif %}
+            </div>
 
-                        <div class="usa-width-one-half va-c-event-share">
-                            <ul class="vads-u-margin-bottom--1 usa-unstyled-list">
-                                {% if addToCalendarLink %}
-                                    <li><i class="va-c-social-icon fas fa-calendar-check"></i>  <a href="{{ addToCalendarLink }}"> Add to Calendar</a></li>
-                                {% endif %}
-                                <li><i class="va-c-social-icon fab fa-facebook"></i>  <a href=""> Share on Facebook</a></li>
-                                <li><i class="va-c-social-icon fab fa-twitter"></i>  <a href=""> Share on Twitter</a></li>
-                            </ul>
-                        </div>
-                    </div>
+            <div class="usa-width-one-half va-c-event-share">
+              <ul class="vads-u-margin-bottom--1 usa-unstyled-list">
+                {% if addToCalendarLink %}
+                  <li>
+                    <i class="va-c-social-icon fas fa-calendar-check"></i>
+                    <a id="add-to-calendar-link" href="{{ addToCalendarLink }}">
+                      Add to Calendar</a>
+                  </li>
```

Comment:
Instead of using a hidden form, you could store the calendar data in this link, like -

```html
<a data-calendar="{{ fieldDate }}"
   id="add-to-calendar-link" 
   href="{{ addToCalendarLink }}">Add to Calendar</a>
```